### ISSUE TITLE
Version 0.8.2 breaks android support

### ISSUE THREAD

**Issue by MaxwellDAssistek** [oleg-st/ZstdSharp#45]
Hello,

After I updated from 0.8.1 to 0.8.2, I started getting the following exception when trying to use the CompressionStream class:

```
System.TypeInitializationException: The type initializer for 'ZstdSharp.Unsafe.Methods' threw an exception.
 ---> System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Runtime.CompilerServices.RuntimeHelpers.AllocateTypeAssociatedMemory(Type type, Int32 size)
   at ZstdSharp.Unsafe.Methods..cctor()
   --- End of inner exception stack trace ---
   at ZstdSharp.SafeCctxHandle.Create()
   at ZstdSharp.Compressor..ctor(Int32 level)
   at ZstdSharp.CompressionStream..ctor(Stream stream, Int32 level, Int32 bufferSize, Boolean leaveOpen)
```

**Comment by oleg-st** [comment]
@teo-tsirpanis Do you know the right way to handle it?

**Comment by oleg-st** [comment]
1) We can call it in .NET 9+ only
2) We can detect Mono runtime in .NET 5..8
https://github.com/dotnet/BenchmarkDotNet/blob/c7ed714a34a4de04f7320b6d971a51b04545bb1c/src/BenchmarkDotNet/Portability/RuntimeInformation.cs#L35

**Comment by oleg-st** [comment]
Fixed in 0.8.3
